app/src/main/res/xml/preferences_settings.xml

```diff
@@ -3,6 +3,16 @@
     <PreferenceCategory
         android:title="@string/settings_account">
 
+        <ListPreference
+            android:key="settings_network"
+            android:title="@string/settings_network"
+            android:summary="@string/settings_network_summary"
+            android:entries="@array/settings_network_entries"
+            android:entryValues="@array/settings_network_entryvalues"
+            android:defaultValue="T"
```

Comment:
Default value maybe should be set to "M"? When the app is first installed, it should be mainnet by default.